data/categories.txt

```diff
@@ -1387,19 +1397,20 @@ eu:4Likore-denda|likore
 sv:Spritaffär|4alkohol
 th:ร้านขายเหล้า
 tr:İçki dükkanı|alkol
-uk:Винний магазин|крамниця спиртних напоїв|4алкоголь
+uk:Винний|Спиртни напої|4алкоголь
 vi:Cửa hàng rượu|rượu
 zh-Hans:烈酒|商店
 zh-Hant:烈酒|1購物|店鋪|商店|雜貨店|便利商店
 el:Κάβα|ποτά|κατάστημα
 sk:Liehoviny|4alkohol
 fa:مشروب فروشی
 
-shop-books
+shop-books|@shop
 en:Bookstore|bookshop|3books|U+1F4D6|U+1F4DA|U+1F4D9|U+1F4D8|U+1F4D7|U+1F4D5
-ru:Книжный магазин|3книги
+ru:Книжный|3книги
 bg:3Книжарница|книги
 ar:مكتبة لبيع الكتب|متجر
+be:Кнігі
 cs:Knihkupectví
 da:Boghandel|boghandler|boglade
 nl:Boekenwinkel|boekwinkel
```

Comment:
nl:Boeken|Boekenwinkel|boekwinkel|boekhandel

If I follow your reasoning, only "Boeken" might be necessary. So it then becomes

nl:Boeken